packages/vats/src/distributeFees.js

```diff
@@ -30,56 +26,28 @@ export function makeTreasuryFeeCollector(zoe, treasuryCreatorFacet) {
 // remainder till the next epoch.
 
 /** @type {BuildFeeDistributor} */
-export function buildDistributor(treasury, bank, epochTimer, params) {
+export async function buildDistributor(
```

Comment:
Please update the comment above.

// A distributor of fees from treasury to the Bank module. Each time the
// epochTimer signals the end of an Epoch, it will ask
// the treasury for the fees that have been collected to date and send that
//  payment to the depositFacet.